tests/rest_api/tests/api_validate/test_api_validate_details.py

```diff
@@ -6,7 +6,7 @@
     ApiValidateDetails,
 )
 
-details_version = re.compile(r"\d\.\d\d\.\d\d\d")
+details_version = re.compile(r"\d\.\d\d\.\d\d")
```

Comment:
Should be r"\d\.\d\d\.\d+"

Reply:
fixed

Reply:
`passed tests:` https://github.com/veraPDF/veraPDF-rest/actions/runs/10134160731